internal/common/consts.go

```diff
@@ -22,6 +22,8 @@ const (
 	HttpProto         = "HTTP"
 	StatusResponse    = "pong"
 	ServiceStatusPass = "passing"
+	ConfigFileName    = "configuration.toml"
+	ConfigV2Stem      = "config/V2/"
```

Comment:
why "V2"? 

core services are using "edgex/core/1.0/

Reply:
@cloudxxx8 Please review the Edinburgh Release -- Configuration Registry Structure document at the bottom of this page
https://wiki.edgexfoundry.org/display/FA/Architecture+Issues+and+Decisions

Reply:
sorry, corrected it